Integrations/Elasticsearch_v2/Elasticsearch_v2.py

```diff
@@ -169,14 +244,20 @@ def fetch_params_check():
     if FETCH_QUERY == '' or FETCH_QUERY is None:
         str_error.append("Query by which to fetch incidents is not configured.")
 
-    if TIME_FORMAT == '' or TIME_FORMAT is None:
+    if (TIME_FORMAT == '' or TIME_FORMAT is None) and TIME_METHOD == 'Simple-Date':
         str_error.append("Time format is not configured.")
 
     if len(str_error) > 0:
         return_error("Got the following errors in test:\nFetches incidents is enabled.\n" + '\n'.join(str_error))
 
 
 def test_general_query(es):
+    """if is_fetch it ticked, this function runs a generay query to Elasticsearch just to make sure we get a response
+        from the FETCH_INDEX.
+
+    Args:
+        es(Elasticsearch): an Elasticsearch object to which we run the test.
+    """
```

Comment:
Split it into a one line of explanation for the header
Test executing a fetch query.

then add the note

Reply:
done - in all test functions